style.css

```diff
@@ -445,29 +508,60 @@ hr {
   left: 72px;
 }
 
-.framework ul {
-  margin-top: 25px;
+.input-one {
+  width: 60%;
+  height: 140px;
+  padding-bottom: 95px;
+  margin-left: 150px;
+  border: 1px solid rgb(252, 252, 252);
+  background-color: rgb(247, 247, 247);
+  margin-top: 15px;
 }
 
-.framework li {
+.cont p {
+  width: 600px;
+  margin-left: 70px;
   font-family: 'Inter', sans-serif;
   font-style: normal;
-  font-weight: 400;
-  font-size: 22px;
-  line-height: 43px;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  color: #fff;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+.cont-btn {
+  margin-left: 150px;
+  width: 11%;
   margin-top: 15px;
+  background-color: #f55800;
+  color: #fff;
+  height: 48px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  border: none;
+}
```

Comment:
- [ ] You have adjusted the padding of this button like the previous reviewer suggested, but it still doesn't look quite right. Please give this button a width of `131px` to be exactly as specified in the Figma design 🙏